src/plugin/file/kml/ui/kmlnodeui.js

```diff
@@ -13,7 +13,7 @@ goog.require('plugin.file.kml.cmd.KMLNodeRemove');
 plugin.file.kml.ui.kmlNodeUIDirective = function() {
   return {
     restrict: 'E',
-    replace: true,
+    replace: false,
```

Comment:
Why was this needed? We generally try to use `replace: true` because it prevents breaking the chain of flex items (the directive element will not have `d-flex`) which can cause layout problems.